glusterd2/store/store.go

```diff
@@ -18,7 +19,10 @@ import (
 )
 
 const (
-	sessionTTL = 30 // used for etcd mutexes and liveness key
+	sessionTTL    = 30              // used for etcd mutexes and liveness key
+	getTimeout    = 5 * time.Second //timeout for store Get operation
```

Comment:
The variable names are self-explanatory, you can remove the comments for timeouts.

Reply:
done